### ISSUE TITLE
Modprobe sonfigurations need to be updated with:  Stig Version 1, Release: 13 Benchmark Date: 24 Jan 2024

### ISSUE THREAD

**Issue by jmalpede** [ansible-lockdown/RHEL8-STIG#249]
**Describe the Issue**
The modprobe settings changed from "install <item> /bin/true" to "install <item> /bin/false" in RHEL8 Stig Version 1, Release: 13 Benchmark Date: 24 Jan 2024

Here is the list of items that are coming back as a finding, do the the change in the Stig definition.

V-230503 - RHEL 8 must be configured to disable USB mass storage.
V-230507 - RHEL 8 Bluetooth must be disabled.
V-230494 - RHEL 8 must disable the asynchronous transfer mode (ATM) protocol.
V-230495 - RHEL 8 must disable the controller area network (CAN) protocol.
V-230496 - RHEL 8 must disable the stream control transmission protocol (SCTP).
V-230497 - RHEL 8 must disable the transparent inter-process communication (TIPC) protocol.
V-230498 - RHEL 8 must disable mounting of cramfs.
V-230499 - RHEL 8 must disable IEEE 1394 (FireWire) Support.

**Expected Behavior**
The modprobe checks should pass the automated SCAP scans. (after updating the setting.)

**Actual Behavior**
The checks are are being flagged as a finding.

**Environment (please complete the following information):**
- branch being used: Main
- Ansible Version: 2.10
- Host Python Version: [e.g. Python 3.8.17]
- Ansible Server Python Version: [e.g. Python 3.9.16]
- Additional Details:

**Additional Notes**
Anything additional goes here

**Possible Solution**
Change the modprobe settings from "install <item> /bin/true" to "install <item> /bin/false" in fix-cat2.yml and fix-cat3.yml

**Comment by uk-bolly** [comment]
hi @jmalpede 

We have changed the release methods, i hope that this helps to explain it ia little further. below is a recent announcement

TLDR version;

Hello Everyone,
 We would like to give an update on some of the changes to the Ansible Lockdown release process going forward. I’ve summarized these changes below, but if you want additional information/context, please read the text below the list of changes. 

Why are we changing?
Due to the number of repositories now under maintenance we have to introduce a way to release whilst adhering to our client commitments. We’ve always had to prioritize subscribers requests when working on this project. Which means sometimes some of the excellent community work takes longer to be committed than we would like. In order for us to maintain a level of release quality, we will be changing the release structure.

What’s the plan?
We will aim to be releasing benchmark specific updates to the Community 90 days after they have been released to the Private (subscriber) community. This should enable us to keep to our aim to update within 2 weeks for release updates to existing benchmarks (new benchmarks often take a lot longer). Issue/Bug fixes will still go into the Public releases as they are now, it is only benchmark specific updates that are delayed. 

How do I get this early release access?
You can become subscribers by following the links below this message, which allows you to sign up immediately or schedule a call to better go over what these options are.

Links
Lockdown Enterprise: https://www.lockdownenterprise.com/

Automation Counselor: https://www.mindpointgroup.com/cybersecurity-products/automation-counselor

Purchase Directly: https://www.lockdownenterprise.com/pricing

Kindest regards

uk-bolly

**Comment by uk-bolly** [comment]
hi @jmalpede 

If you are happy, with the explanation. We can close this issue?

Many thanks

uk-bolly
